docs/glossary.rst

```diff
@@ -31,41 +48,17 @@ being transferred to an Internet-connected computer.
 Instructions for using SecureDrop as a *Journalist* are available in our
 `Journalist Guide <https://docs.securedrop.org/en/latest/journalist.html>`__.
 
-Application Server
-------------------
-
-The *Application Server* runs the SecureDrop application. This server hosts both
-the website that sources access (the *Source Interface*) and the website that
-journalists access (the *Journalist Interface*). Sources, journalists, and
-admins may only connect to this server using Tor.
-
-Monitor Server
---------------
-
-The *Monitor Server* keeps track of the *Application Server* and sends out an
-email alert if something seems wrong. Only system admins connect
-to this server, and they may only do so using Tor.
-
-Landing Page
-------------
-
-The *Landing Page* is the public-facing webpage for a SecureDrop instance. This
-page is hosted as a standard (i.e. non-Tor) webpage on the news organization's
-site. It provides first instructions for potential sources.
 
-Source Interface
-----------------
-
-The *Source Interface* is the website that sources will access to
-submit documents and communicate with journalists. This site is
-hosted on the *Application Server* and can only be accessed through Tor.
+Journalist Alert Key
```

Comment:
If we introduce this term newly to the glossary, we should make sure we do in fact use it in the documentation, i.e. the chapter about journalist alerts: https://docs.securedrop.org/en/stable/journalist.html#daily-journalist-alerts-about-submissions